### ISSUE TITLE
Online Readme.md doesn't have accurate flag descriptions

### ISSUE THREAD

**Issue by DrWhy44** [007revad/Synology_HDD_db#261]
![image](https://github.com/007revad/Synology_HDD_db/assets/155069669/31688e20-d922-4b2b-9b0f-39d31f8299c7)

**Comment by 007revad** [comment]
You cropped the important part of the script's output: The script version.

Your left half part of your image shows an older version of the syno_hdd_db script while the readme is up to date for the latest version of the script.

I can tell that you are at least 6 versions behind the [latest version](https://github.com/007revad/Synology_HDD_db/releases).

**Comment by DrWhy44** [comment]
your right, i thought I ran the autoupdate flag prior to doing this, but apparently not. what happened to the M command?  Does the m2 flag not exist anymore?

**Comment by 007revad** [comment]
I originally added the -m flag just in case people were worried that the script might mess up their existing NVMe volumes manually created in DSM 6 or 7.0. 

But then I was getting too many questions about what the -m flag was for. I also suspected nobody used it, and did a survey on reddit asking if anyone uses the -m flag and got no replies. 

So I removed the -m flag from the readme and help but left it in the script just in case anyone actually does use it.
